dcrtimed/dcrtimed.go

```diff
@@ -1201,6 +1233,77 @@ func (d *DcrtimeStore) lastAnchorV2(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+func (d *DcrtimeStore) lastDigestsV2(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+
+	var ld v2.LastDigests
+	decoder := json.NewDecoder(r.Body)
+	decoder.DisallowUnknownFields()
+	if err := decoder.Decode(&ld); err != nil {
+		util.RespondWithError(w, http.StatusBadRequest,
+			"Invalid request payload")
+		return
+	}
+
+	log.Infof("%v LastDigests %v", r.URL.Path, r.RemoteAddr)
+
+	ldr, err := d.backend.LastDigests(ld.N)
+	if err != nil {
+		errorCode := time.Now().Unix()
```

Comment:
Handling errors this way will lead to a lot of duplicate and boiler plate code in each route.  You may want to abract this out.  It will make the code cleaner and easier to read.  Example:

https://github.com/decred/politeia/blob/master/politeiawww/legacy/comments/error.go#L21
https://github.com/decred/politeia/blob/master/politeiawww/legacy/comments/comments.go#L42-L70